source/mongodump.txt

```diff
@@ -372,6 +372,21 @@ FIPS
 connections to a :binary:`~bin.mongod`/:binary:`~bin.mongos` that is
 :manual:`configured to use FIPS mode </tutorial/configure-fips>`.
 
+Using ``mongodump`` on Atlas Cluster 
```

Comment:
[suggestion] This applies to the mongorestore page, too, but for grammar, I think we either want to drop "cluster," make it plural into "Atlas Clusters," or add "an" before "Atlas Cluster".

```suggestion
Using ``mongodump`` on Atlas Clusters 
```